### PR TITLE
Armstrong number code in C# added

### PR DESCRIPTION
**Pull request by sharur7**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

In recreational number theory, a narcissistic number is also known as a pluperfect digital invariant (PPDI), an Armstrong number or a plus perfect number is a number that is the sum of its digits each raised to the power of the number of digits.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1217

### CODE REVIEW

**Comment by sharur7** [comment]
@iamrajiv @SKAUL05 @Kajol-Kumari  Can you please review my PR?

**Review by SKAUL05 — APPROVED** [review]

**Review by iamrajiv — APPROVED** [review]

File: C-Sharp/math/Armstrong_Number.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Armstrong Number
+using System;
+public class Armstrong // Class Declaration
+{
+  public static void Main(string[] args) {
+    int number,
+    r,
+    sum = 0,
+    temp;
+    Console.Write("Enter the Number to check whether it is Armstrong Number or Not = ");
```
Comment (atarax665) [inline C-Sharp/math/Armstrong_Number.cs]:
```suggestion
    Console.Write("Enter the Number to check whether it is an Armstrong Number or Not = ");
```

File: C-Sharp/math/Armstrong_Number.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    number = int.Parse(Console.ReadLine()); // Getting Number From User    
+    temp = number;
+    while (number > 0) {
+      r = number % 10;
+      sum = sum + (r * r * r);
+      number = number / 10;
+    }
+    // Checking if the number is Armstrong number or not
+    if (temp == sum) Console.Write("{0} Is Armstrong Number.", temp);
```
Comment (atarax665) [inline C-Sharp/math/Armstrong_Number.cs]:
```suggestion
    if (temp == sum) Console.Write("{0} Is an Armstrong Number.", temp);
```

File: C-Sharp/math/Armstrong_Number.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    else Console.Write("{0} Is Not Armstrong Number.", temp);
```
Comment (atarax665) [inline C-Sharp/math/Armstrong_Number.cs]:
```suggestion
    else Console.Write("{0} Is Not an Armstrong Number.", temp);
```

File: C-Sharp/math/Armstrong_Number.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
+// Armstrong Number
+using System;
+public class Armstrong // Class Declaration
+{
+  public static void Main(string[] args) {
+    int number,
```
Comment (atarax665) [inline C-Sharp/math/Armstrong_Number.cs]: Declare all the variables in one line. 
ex - ```int number, r, sum = 0, temp;```

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: C-Sharp/math/Armstrong_Number.cs (hunk 1 above)
Comment (sharur7) [reply C-Sharp/math/Armstrong_Number.cs]: done

File: C-Sharp/math/Armstrong_Number.cs (hunk 2 above)
Comment (sharur7) [reply C-Sharp/math/Armstrong_Number.cs]: done

File: C-Sharp/math/Armstrong_Number.cs (hunk 3 above)
Comment (sharur7) [reply C-Sharp/math/Armstrong_Number.cs]: done

File: C-Sharp/math/Armstrong_Number.cs (hunk 4 above)
Comment (sharur7) [reply C-Sharp/math/Armstrong_Number.cs]: done

**Review by atarax665 — APPROVED** [review]

**Comment by atarax665** [comment]
@sharur7 Resolve merge conflict to get this PR merged ASAP.

**Comment by sharur7** [comment]
> @sharur7 Resolve merge conflict to get this PR merged ASAP.

@atarax665 Done

**Pull request merged**